src/client/Client.cc

```diff
@@ -3530,8 +3530,7 @@ int Client::get_caps_used(Inode *in)
 void Client::cap_delay_requeue(Inode *in)
 {
   ldout(cct, 10) << __func__ << " on " << *in << dendl;
-  in->hold_caps_until = ceph_clock_now();
-  in->hold_caps_until += cct->_conf->client_caps_release_delay;
+  in->hold_caps_until = ceph::coarse_mono_clock::now() + cct->_conf.get_val<std::chrono::seconds>("client_caps_release_delay");
```

Comment:
give a line break

Reply:
Please break this into two lines, please see the code style doc in https://github.com/ceph/ceph/blob/master/CodingStyle and https://google.github.io/styleguide/cppguide.html#Line_Length

Reply:
**cct->_conf.get_val<std::chrono::seconds>("client_caps_release_delay")** replaced with **caps_release_delay**.Now its less than 80 chars.